### PR TITLE
feat(single-handle-slider): Export Slider

### PR DESCRIPTION
**Pull request by jes708**
Export { Slider } and RangeSlider from pcln-slider

This PR mostly moves some files around for better organization, and exports Slider, which already existed prior to this PR but wasn't being exported.  This PR also updates docs to reflect the new Slider export.

### CODE REVIEW

**Comment by bertya** [comment]
LGTM, just need to double-check our current release process that should we update version here or after merge

**Comment by jes708** [comment]
> LGTM, just need to double-check our current release process that should we update version here or after merge

I think you may be right.  I will revert the versions, and take care of that on publish.

File: packages/slider/src/RangeSlider.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Range from './rc-slider/Range'
+import { space, color } from 'styled-system'
+import styleSlider from './styleSlider'
+
+const RangeSlider = styleSlider(Range)
+
+RangeSlider.defaultProps = {
+  allowCross: false,
+  color: 'blue'
```
Comment (James300) [inline packages/slider/src/RangeSlider.js]: This should be `primary.base`

File: packages/slider/src/RangeSlider.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+RangeSlider.propTypes = {
+  ...space.propTypes,
+  ...color.propTypes
```
Comment (James300) [inline packages/slider/src/RangeSlider.js]: I don't think we should be using `color` from styled-system anymore

File: packages/slider/src/RangeSlider.js (hunk 1 above)
Comment (jes708) [reply packages/slider/src/RangeSlider.js]: Can that be done in a follow-up PR? This version of pcln-slider does not require DS 2.3.1 yet.

File: packages/slider/src/RangeSlider.js (hunk 2 above)
Comment (jes708) [reply packages/slider/src/RangeSlider.js]: This version of pcln-slider does not require DS 2.3.1 yet, so it should be ok for now right?

File: packages/slider/src/RangeSlider.js (hunk 1 above)
Comment (James300) [reply packages/slider/src/RangeSlider.js]: Sounds good to me

**Pull request merged**